sql/core/src/test/resources/sql-tests/inputs/extract.sql

```diff
@@ -1,134 +1,3 @@
-CREATE TEMPORARY VIEW t AS select '2011-05-06 07:08:09.1234567' as c, interval 10 year 20 month 30 day 40 hour 50 minute 6.7890 second as i;
```

Comment:
The view `t` is no longer created if `spark.sql.legacy.interval.enabled` is `false` and the queries which refer to `t` always fail.
So I moved this part to `inputs/legacy_interval/extract.sql`.